### PR TITLE
[redox_w] add dvorak layout and white led support

### PR DESCRIPTION
**Pull request by 66f94eae**
## Description
add white led control,and remove useless led define,like "set_led_yellow"  @mattdibi 
add dvorak layout,and caps_lock,num_lock led Indicators

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/redox_w/keymaps/dvorak/keymap.c [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+#include QMK_KEYBOARD_H
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _DVORAK 0
+#define _SYMB 1
+#define _QWERTY 2
+#define _ADJUST 3
```
Comment (drashna) [inline keyboards/redox_w/keymaps/dvorak/keymap.c:26]:
```suggestion
enum layers {
    _DVORAK,
    _SYMB,
    _QWERTY,
    _ADJUST,
};
```

File: keyboards/redox_w/redox_w.h [hunk 2]
```diff
@@ -8,15 +8,14 @@
...
+#define wht_led_on    PORTD &= ~(1<<0)
 
-#define set_led_off     red_led_off; grn_led_off; blu_led_off
-#define set_led_red     red_led_on;  grn_led_off; blu_led_off
-#define set_led_blue    red_led_off; grn_led_off; blu_led_on
-#define set_led_green   red_led_off; grn_led_on;  blu_led_off
-#define set_led_yellow  red_led_on;  grn_led_on;  blu_led_off
-#define set_led_magenta red_led_on;  grn_led_off; blu_led_on
-#define set_led_cyan    red_led_off; grn_led_on;  blu_led_on
-#define set_led_white   red_led_on;  grn_led_on;  blu_led_on
```
Comment (drashna) [inline keyboards/redox_w/redox_w.h:19]: why are these removed?

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/redox_w/redox_w.h (hunk 2 above)
Comment (66f94eae) [reply keyboards/redox_w/redox_w.h:19]: As you can see, the keyboard contains only 4 LEDs (default D1: red, D2: blue, D3: grene, D4: white). 
Each led has its own switch, not RGB style, which means that white led on does not mean red and blue and green are on at the same time. 
So I think these define is useless.
led_pic

File: keyboards/redox_w/redox_w.c [hunk 3]
```diff
@@ -1,8 +1,8 @@
 #include "redox_w.h"
 
 void led_init(void) {
-	DDRD  |= (1<<1);
-	PORTD |= (1<<1);
+	DDRD  |= (1<<0) | (1<<1);
```
Comment (drashna) [inline keyboards/redox_w/redox_w.c:4]: ideally, these should be changed to GPIO commands. 

https://docs.qmk.fm/#/internals_gpio_control

Comment (66f94eae) [reply keyboards/redox_w/redox_w.c]: Ok, the modification has been completed

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!
